styles/readwise/catppuccin.user.css

```diff
@@ -0,0 +1,227 @@
+      );
+      content: url("data:image/svg+xml,@{svg}");
+    }
+
+    .streak-icon {
+      @svg: escape(
+        '<svg width="39" height="49" viewBox="0 0 39 49" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M26.2 7.775 9.376 24.488l8.949 3.061-5.583 13.07 16.825-16.713-8.949-3.062z" fill="@{accent-color}"/></svg>'
+      );
+      content: url("data:image/svg+xml,@{svg}");
+    }
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
```

Comment:
Can we keep these "base styles" together with/below the `color-scheme` rule above on line 70?